src/test/java/org/fundacionjala/coding/christian/Kata6Test.java

```diff
@@ -0,0 +1,62 @@
+package org.fundacionjala.coding.christian;
+
+import org.junit.Before;
+import org.junit.Test;
+import java.util.Arrays;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Powered by Christian Galarza.
+ */
+public class Kata6Test {
+    private Kata6 sort;
+
+    /**
+     *
+     */
+    @Before
+    public void setup() {
+        sort = new Kata6();
+    }
+
+    private final Integer[] kepler1 = new Integer[]{1, 2, 7, 4, 5, 6, 3, 8, 9};
+    private final Integer[] earth1 = new Integer[]{1, 2, 3, 4, 5, 6, 7, 8, 9};
+    private final Integer[] kepler2 = new Integer[]{12, 14, 13};
+    private final Integer[] earth2 = new Integer[]{12, 13, 14};
+    private final Integer[] kepler3 = new Integer[]{2, 7, 4, 3, 9};
+    private final Integer[] earth3 = new Integer[]{9, 2, 4, 7, 3};
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1170)